Makefile

```diff
@@ -9,9 +13,7 @@ LISTEN_ADDRESS ?= 0.0.0.0:0
 LEDGER_DIR ?= ./ledger.db
 CHAIN_DIR ?= ./chain.db
 
-.PHONY: help bootstrap run import-snapshots import-headers import-nonces download-haskell-config  coverage-html coverage-lconv check-llvm-cov
-
-all: help
+.PHONY: help bootstrap run import-snapshots import-headers import-nonces download-haskell-config coverage-html coverage-lconv check-llvm-cov
```

Comment:
Accident or done on purpose?

Reply:
`make` already uses uses the first target as [default](https://www.gnu.org/software/make/manual/html_node/How-Make-Works.html)